server/database/createTable.sql

```diff
@@ -166,12 +166,12 @@ $$;
 CREATE OR REPLACE FUNCTION check_property() RETURNS TRIGGER AS $$ BEGIN IF EXISTS (
         SELECT 1
         FROM task_property
-        WHERE id = NEW.task_id
-            AND name = NEW.name
+        WHERE task_id = NEW.task_id
```

Comment:
This was very important to fix property behavior on the back end most of the changes in this file are to correct latent bugs with properties and with functions. as well as with adding test tasks corrected to not force and ID on them.